scriptworker/cot/verify.py

```diff
@@ -1030,11 +1034,61 @@ def verify_link_in_task_graph(chain, decision_link, task_link):
     }
 
 
-def _get_additional_github_releases_jsone_context(parent_link, decision_link):
+async def _get_additional_github_releases_jsone_context(decision_link):
+    context = decision_link.context
+    source_env_prefix = context.config['source_env_prefix']
+    task = decision_link.task
+    repo_url = get_repo(task, source_env_prefix)
+    repo_owner, repo_name = extract_github_repo_owner_and_name(repo_url)
+    tag_name = get_revision(task, source_env_prefix)
+
+    release_data = GitHubRepository(
+        repo_owner, repo_name, context.config['github_oauth_token']
+    ).get_release(tag_name)
+
+    # The release data expose by the API[1] is not the same as the original event[2]. That's why
+    # we have to rebuild the object manually
+    #
+    # [1] https://developer.github.com/v3/repos/releases/#get-a-single-release
+    # [2] https://developer.github.com/v3/activity/events/types/#releaseevent
+    return {
+        'event': {
+            'repository': {
+                # TODO: Append ".git" to clone_url in order to match what GitHub actually provide.
+                # This can't be done at the moment because some mobile projects still rely on the
+                # bad value
```

Comment:
We should move to taskgraph everywhere; that may provide an opportunity to standardize stuff like this.
You have a similar comment below; should we call a noop function that we'll add `.git` to later? Non-blocking.

Reply:
Okay. I'll address it in a followup